src/cli.js

```diff
@@ -593,6 +600,12 @@ var exports = {
     var results = [];
     var data = [];
 
+    function mergeCLIPrereq(config) {
+      if (opts.prereq) {
+        config.prereq = (config.prereq || []).concat(opts.prereq.split(/\s*,\s*/));
```

Comment:
@jugglinmike 
this only supports whitespace where it has already been escaped. e.g. In my tests..

```
$ jshint a.js --prereq b.js ,c.js
```
will try and open `,c.js` but
```
$ jshint a.js --prereq "b.js, c.js"
```
and
```
$ jshint a.js --prereq b.js,\ c.js
```
will treat it as one argument.

So I think your comment is already resolved through pre-processing before it gets into this code.

Are you happy with that? I've resolved the other code comments.